catppuccin_options_tmux.conf

```diff
@@ -0,0 +1,47 @@
+# All options and their defaults.
+#
+# This is executed separately to the main configuration
+# so that options are set before parsing the rest of the config.
+
+set -ogq @catppuccin_flavor "mocha"
+
+set -ogq @catppuccin_status_background "default"
+
+# Menu styling options
+set -ogq @catppuccin_menu_style "default"
+set -ogq @catppuccin_menu_selected_style "fg=#{@thm_surface_0},bg=#{@thm_yellow}"
+set -ogq @catppuccin_menu_border_style "default"
+
+# Pane styling options
+set -ogq @catppuccin_pane_status_enabled "no" # set to "yes" to enable
+set -ogq @catppuccin_pane_border_status "off" # set to "yes" to enable
+set -ogq @catppuccin_pane_border_style "fg=#{@thm_overlay_0}"
+set -ogq @catppuccin_pane_active_border_style "#{?pane_in_mode,fg=#{@thm_lavender},#{?pane_synchronized,fg=#{@thm_magenta},fg=#{@thm_lavender}}}"
+set -ogq @catppuccin_pane_left_separator "█"
+set -ogq @catppuccin_pane_middle_separator "█"
+set -ogq @catppuccin_pane_right_separator "█"
+set -ogq @catppuccin_pane_color "#{@thm_green}"
+set -ogq @catppuccin_pane_background_color "#{@thm_surface_0}"
+set -ogq @catppuccin_pane_default_text "#{b:pane_current_path}"
+set -ogq @catppuccin_pane_default_fill "number"
+set -ogq @catppuccin_pane_number_position "left" # right, left
+
+# Window options
+set -ogq @catppuccin_window_status_style "basic" # basic, rounded, slanted, custom, or none
+set -ogq @catppuccin_window_default_fill "number"
+set -ogq @catppuccin_window_default_color "#{@thm_bg}"
+set -ogq @catppuccin_window_default_background "#{@thm_overlay_2}"
+set -ogq @catppuccin_window_default_text " ##T"
+set -ogq @catppuccin_window_current_fill "number"
+set -ogq @catppuccin_window_current_color "#{@thm_surface_1}"
+set -ogq @catppuccin_window_current_background "#{@thm_peach}"
+set -ogq @catppuccin_window_current_text " ##T"
+set -ogq @catppuccin_window_number_position "left"
```

Comment:
Maybe change `@catppuccin_window_current_text`/`@catppuccin_window_text` to `#W`/`#{window_name}` it's the default for tmux and there is almost every other week an issue/comment about it.

Reply:
This is similar to the default, but it means applications can set the window title themselves. neovim in particular utilises this, as do fish and oh-my-zsh. I agree that the current directory is a poor default. I'm happy to change this to `#W` if desired, but `#T` has been an amazing default for me